module-apps/apps-common/locks/handlers/PhoneLockHandler.hpp

```diff
@@ -20,6 +20,13 @@ namespace locks
 
     class PhoneLockHandler
     {
+      public:
+        enum class TypeOfRequest
+        {
+            Normal = 0,
+            Soft
```

Comment:
why do we have two types of phone lock?

Reply:
Applies to the scenario where the user plugs in the USB cable when there is already a pop-up on the screen to enter the password.
This scenario was mentioned by QA ["After connecting Pure to the computer, while entering the passcode,[...]](https://appnroll.atlassian.net/browse/MOS-686?focusedCommentId=67376)